test/src/main/java/org/apache/accumulo/test/shell/ShellServerIT.java

```diff
@@ -1460,7 +1460,9 @@ public void listscans() throws Exception {
       SlowIterator.setSleepTime(cfg, 500);
       s.addScanIterator(cfg);
 
-      Thread thread = new Thread(() -> Iterators.size(s.iterator()));
+      Thread thread = new Thread(() -> {
+        assertTrue(Iterators.size(s.iterator()) > 0);
+      });
```

Comment:
This is a weird one, because the assertion is happening inside a thread. I'm not sure that will fail visibly. Could do:

```suggestion
      var iterSize = new AtomicInteger(0);
      Thread thread = new Thread(() -> iterSize.set(Iterators.size(s.iterator())));
      assertTrue(iterSize.get() > 0);
```